tests/writer_test.py

```diff
@@ -59,9 +60,14 @@ def testYamlWriter(self):
     file_object = io.StringIO(initial_value=artifacts_yaml)
     converted_artifact_definitions = list(artifact_reader.ReadFileObject(
         file_object))
-    self.assertCountEqual(
-        [artifact.AsDict() for artifact in artifact_definitions],
-        [artifact.AsDict() for artifact in converted_artifact_definitions])
+    if sys.version_info >= (3, 0):
```

Comment:
Python2 assertItemsEqual is renamed assertCountEqual in Python3.  I am open to better solutions as this was the simplest fix I could think of when fighting getting the unit tests to pass in 2 and 3.